packages/cli/src/commands/repo/lint.ts

```diff
@@ -30,9 +36,44 @@ function depCount(pkg: BackstagePackageJson) {
   return deps + devDeps;
 }
 
+const CACHE_FILE_NAME = 'lint-cache.json';
+
+type Cache = string[];
```

Comment:
not worth worrying too much about, but it'd help understanding the on-disk format if it were like

```json
{
  "version": 1,
  "successfullyLintedTreeHashes": [...]
```

or whatnot.